### PR TITLE
feat/connect-be: mock BE using the api

### PR DESCRIPTION
**Pull request by stavros-liaskos**
All BE endpoints are listed in openapi.yml

### CODE REVIEW

File: components/List/List.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const List: React.FunctionComponent<ListProps> = ({ list, i18n }) => {
 
   const followArtist = (artistData: ListEl) => {
     setArtistLoading(artistData.name);
-    fetch('/me/unfollow', {
+    fetch('https://release-racconBE.com/todos/me/unfollow', {
```
Comment (stavros-liaskos) [inline components/List/List.tsx:18]: use real url

File: components/Main/Main.tsx [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
   const { user, error, isLoading } = useUser();
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const [list, setList] = useState<ListEl[]>(
-    !defaultList.length ? mockedResponse.artistsPerResource.fromLastfm : defaultList,
-  ); // todo rm hardcoded data, get scrapped from db
+  const [list, setList] = useState<ListEl[]>();
+
+  useEffect(()=>{
+    fetch('https://release-racconBE.com/todos/me/myArtists', {
```
Comment (stavros-liaskos) [inline components/Main/Main.tsx:18]: use real url

File: components/Search/Search.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ const Search: React.FunctionComponent<SearchProps> = ({ i18n }) => {
   const handleFollow = (artistData: ListEl) => {
     setDisabled(true);
 
-    fetch('/me/follow', {
+    fetch('https://release-racconBE.com/todos/me/follow', {
```
Comment (stavros-liaskos) [inline components/Search/Search.tsx:36]: use real url

File: components/List/List.tsx (hunk 1 above)
Comment (jaivalis) [reply components/List/List.tsx:18]: Could we not make the base url configurable and refer to it here instead? Same goes for the below place holder comments

**Pull request merged**